config/config.go

```diff
@@ -52,6 +52,9 @@ var (
 
 	defaultNodeKeyPath  = filepath.Join(defaultConfigDir, defaultNodeKeyName)
 	defaultAddrBookPath = filepath.Join(defaultConfigDir, defaultAddrBookName)
+
+	minSubscriptionBufferSize     = 100
+	defaultSubscriptionBufferSize = 1000
```

Comment:
Sure, we can drop this to a more modest default value. How many subscribers do people usually see in a full node or validator though?

Reply:
> Sure, we can drop this to a more modest default value. How many subscribers do people usually see in a full node or validator though?

I don't know—I suspect it depends a lot on the size of the network and the behaviour of the application. I mainly want to avoid blowing up nodes that didn't already need this new knob.